k8s/client.go

```diff
@@ -492,6 +492,19 @@ func (c *Client) CiliumStatus(ctx context.Context, namespace, pod string) (*mode
 	return &statusResponse, nil
 }
 
+func (c *Client) CiliumDbgEndpoints(ctx context.Context, namespace, pod string) ([]*models.Endpoint, error) {
+	stdout, err := c.ExecInPod(ctx, namespace, pod, defaults.AgentContainerName, []string{"cilium", "endpoint", "list", "-o", "json"})
```

Comment:
Do we need a version switch here? For <= 1.14, `cilium` and for > 1.14, `cilium-dbg`?

Reply:
Hmm, fair point. I honestly just adopted the same approach as `CiliumStatus` above, which always uses `cilium`. And it doesn't create problems because newer versions still ship a `cilium` soft link to the `cilium-dbg` binary. I looked through the repo, and it seems that there are no usages of `cilium-dbg` (while the `cilium` binary is used quite a lot), so I'd propose to keep it this way for the moment.